superset/templates/appbuilder/footer.html

```diff
@@ -0,0 +1,8 @@
+<div class="navbar navbar-default navbar-fixed-bottom navbar-template navbar-default-template navbar-fixed-bottom-template">
+    <div class="footer-left">
+      <label class='footer-left-label'>{{ appbuilder.get_app.config['TIMEZONE'] }}</label>
```

Comment:
👍 
I would like if we can create template variables as done in navbar.html
```
 {% set timezone = appbuilder.app.config.get('TIMEZONE')%}
```

get handles exception better than box brackets

Reply:
done,

1. currently we are defining TIMEZONE and COPYRIGHT variables in  config.py  file ,but these can be override from superset_config.py if required at the time of deployement
2. TIMEZONE is defined in config as constant because of  it is not available in userinfo